vue-ui/src/router/index.js

```diff
@@ -10,21 +29,21 @@ const router = createRouter({
       component: HomeView,
     },
     {
-      path: '/about',
-      name: 'about',
-      // route level code-splitting
-      // this generates a separate chunk (About.[hash].js) for this route
-      // which is lazy-loaded when the route is visited.
-      component: () => import('../views/AboutView.vue'),
-    },
-    {
-      path: '/test-chart',
-      name: 'test',
+      path: '/south-bird-island-chart',
```

Comment:
Not a "must do" and up to the team, but I suggest using only `south-bird-island` for the route if we don't have anything else for it beside the chart. `chart` seems to be redundant. 
No change needed now, you can `Resolve Conversation` after you read it as acknowledgement of my comment